tutorials/_Quick_start.md

```diff
@@ -0,0 +1,78 @@
+# Quick Start guide 
+This quick start guide is based on the other tutorials. It is only directly applicable if you have access to the HPC PIK.
+* If you are a PIK member not yet set up on the HPC, look at these [instructions from RSE](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Cluster-Access)*.
+  
+# 1. Getting the code
+*I recommend VScode but any terminal with git will be fine*. *If you do not have git set up and don't know how to do it see the sparse instructions at [RSE git](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Git-Setup) or ask chat gpt "how can I setup git in \[vscode | wsl | windows\] and connect my github account".*
+
+- Fork the [REMIND repo](https://github.com/remindmodel/remind/tree/develop) to your github account
+	- You can uncheck "develop only" but we will fix this below
+- clone the code (instead of a full github clone and checkout)
+	-  `cloneremind https://github.com/<yourusername>/remind.git DESTINATION_FOLDER`
+	- why? this will keep the data references from the HPC and reduce the size of the operation
+- Check the state of your repo
+	- navigate to your repo `cd myrepo`
+	- `git remote -v`
+- Switch to a stable release
+	- run `git fetch --all --tags` if you forgot to get the tags
+	- switch to a (hopefully) e.g. stable release `git checkout tags/v3.4.0 -b release-3.4.0`
+
+ See tutorial [01](https://github.com/remindmodel/remind/blob/develop/tutorials/01_GettingREMIND.md) for more details.
+ 
+# 2. Preliminaries
+
+## Overview of Remind code
+Read [REMIND - REgional Model of INvestments and Development – Overview](https://rse.pik-potsdam.de/doc/remind/3.2.0/) or equivalently the comment starting `main.gams` . This will be helpful in terms of understanding the code.
```

Comment:
```suggestion
Read [REMIND - REgional Model of INvestments and Development – Overview](https://rse.pik-potsdam.de/doc/remind/3.2.0/) or equivalently the comment starting `main.gms`. This will be helpful in terms of understanding the code.
```